src/FlashcardSystem.java

```diff
@@ -1,2 +1,122 @@
+import java.util.*;
+
 public class FlashcardSystem {
+    public DeckController deckController = new DeckController();
+    public SessionController sessionController = new SessionController();
+    public Scanner scanner = new Scanner(System.in);
+
+    public void displayMainMenu() {
+        System.out.println("(0) Logout (1) Create a Deck (2) Select deck to study (3) Edit deck");
+        String select = scanner.nextLine();
+        //TODO: check for invalid input
+        switch(select) {
+            case "0":
+                break;
+            case "1":
+                displayCreateDeckMenu();
+                break;
+            case "2":
+                displaySelectDeckToStudyMenu();
+                break;
+            case "3":
+                displayEditDeckMenu();
+                break;
+        }
+
+    }
+
+    public Deck displayDecks() {
+        //TODO: change format to (0) Back (1) Deck 1 (2) Deck 2 ....
+        List<Deck> decks = deckController.decks;
+        for (int i=0; i<decks.size(); i++) {
+            System.out.println("("+i+") Deck "+i);
+        }
+        String x = scanner.nextLine();
+        //TODO: handle NumberFormatException
+        return decks.get(Integer.parseInt(x));
+    }
+
+    public Flashcard displayCards(Deck deck) {
+        //TODO: change format to (0) Back (1) Card 1 (2) Card 2 ....
+        List<Flashcard> flashcards = deck.getFlashcards();
+        for (int i=0; i<flashcards.size(); i++) {
+            System.out.println("("+i+") Card "+i);
```

Comment:
```suggestion
            System.out.println("("+i+") Card "+ flashcards.get(i).getFront());
```
Ditto here, it'd be nice to be able to identify cards by their front.